docs/prover/checking/sanity.md

```diff
@@ -114,6 +112,10 @@ rule tautology {
 Since every `uint` satisfies the assertion, the assertion is tautological, which
 may indicate an error in the specification.
 
+The tautology check will add a node per `assert` or `satisfy` statement to each rule.  The nodes are named with the prefix `assert_not_tautological_<ID>`.  For example, the rule report for the above `tautology` rule will look like this:
```

Comment:
nit - `<ID>` isn't an ID, but the `<line number>_<start_column>` of the expression. tioli if you think this detail is required in the docs.

Reply:
Changed everywhere to `<LINE>_<COL>` as a middle ground that explains it but is still shorter